### ISSUE TITLE
Dashboard: Template insertion broken

### ISSUE THREAD

**Issue by swissspidy** [swissspidy/web-stories-wp#12345]
## Bug Description

PR #12237 removed the elements registration from the dashboard app because it was deemed unused and thus redundant.

Turns out the elements are in fact needed when using the "Create story from template" functionality, because there we generate the AMP HTML output (via the `output` package) which in turn requires elements to be registered.

While we would have had an e2e test for this scenario that should have caught this right away, it has been disabled because of flakiness (see #12024)

The quick solution here is to revert #12237, the best solution would be to not use the `output` package at all since we don't actually need it.

## Expected Behaviour

Creating a new story from a template should be possible again.

## Steps to Reproduce

1. Go to "Explore Templates"
2. Try to create a new story from a template.

## Additional Context

- Plugin Version: 1.25.0
- WordPress Version: 6.0.2
- Operating System: macOS
- Browser: Chrome

**Comment by kkalarickal** [comment]
@swissspidy - On # 12346 code base, I confirmed the simple workflow of creating a story from template.

Also checked the path for saving page templates and using them.

@swissspidy - Since you removed #12237, I also did a sanity check on stickers functionality since you mentioned the dependency on elements-library which i assume was re-introduced here. Let me know if there is any thing that that needs to be checked because of the revert.
